support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx

```diff
@@ -121,11 +124,15 @@ export function TsAndCsFooterLinks({
 
 	const terms = amountIsAboveThreshold
 		? termsSupporterPlus('Terms and Conditions')
+		: inAdLite
+		? termsGuardianAdLite('Terms')
 		: termsContributions;
+	const productNameSummary = inAdLite ? ' the Guardian Ad-Lite ' : ' our ';
 
 	return (
 		<div css={marginTop}>
-			By proceeding, you are agreeing to our {terms}.{' '}
+			By proceeding, you are agreeing to{productNameSummary}
```

Comment:
why not put the leading space in here rather than in the `productNameSummary` string?

Reply:
Yes, will avoid spaces in string consts going forwards. Updated.